cfgov/jinja2/v1/youth_employment_success/goals.html

```diff
@@ -90,7 +91,7 @@ <h1 class="h1">Planning how you’ll get to work</h1>
     {{
       textarea.render({
         'fieldClass': 'js-goal-steps',
-        'helperText': 'Now work backwards — what are some steps you can take to achieve your goal by this time? Make sure your steps are realistic and specific. For example, my first step is “Over the next six months, I’ll take the bus instead of using a rideshare in order to save money. I am trying to save $400 during this time to put towards a car down payment.”',
+        'helperText': 'ow work backwards — what are some steps you could take to achieve your goal during your intended timeline? Make sure your steps are realistic and specific. For example, my first step is “Over the next six months, I’ll take the bus instead of using a rideshare in order to save money. I’m trying to save $400 over these six months to put towards a car down payment.”',
```

Comment:
`ow work backwards`?

Reply:
Yep, it was a typo, should be updated now!